### PR TITLE
Typing cleanup

### PR DESCRIPTION
**Pull request by JedMeister**
More turd polishing...

### CODE REVIEW

**Review by Christie018 — APPROVED** [review]
Code changes look good to me 😃

File: sysversion/__init__.py [hunk 1]
```diff
@@ -10,98 +10,99 @@
...
-    if rootfs and rootfs != '/':
+    if rootfs != '/':
         comm = ['chroot', rootfs] + comm
-    proc = subprocess.run(comm, stdout=PIPE,
-                          encoding=encoding)
+    proc = subprocess.run(comm, capture_output=True, text=True)
     if proc.returncode != 0:
-        return
-    return proc.stdout.strip()
+        raise TurnkeyVersionError("lsb_release failed: {proc}'")
```
Comment (OnGle) [inline sysversion/__init__.py]: fstring missing the f

**Review by OnGle — APPROVED** [review]
lgtm!

**Review by OnGle — CHANGES_REQUESTED** [review]
My mistake, accidently approved with "lgtm" but that f-string should be fixed

File: sysversion/__init__.py (hunk 1 above)
Comment (JedMeister) [reply sysversion/__init__.py]: Good catch, thanks!

**Comment by JedMeister** [comment]
Issue fixed. Merging...

**Pull request merged**